rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pipeline-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: pipeline_alerts
+      interval: 1m
+      rules:
+        - alert: HighSchedulingOverhead
+          expr: |
+            (sum(sum_over_time(pipelinerun_duration_scheduled_seconds_sum{status='succeded'}[30m]))
+            /
+            sum(sum_over_time(pipelinerun_duration_scheduled_seconds_count{status='succeded'}[30m])))
+            /
+            (sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
+            /
+            sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m])))  > 0.05
```

I think indenting the brackets might make it more readable:
```suggestion
            (
                sum(sum_over_time(pipelinerun_duration_scheduled_seconds_sum{status='succeded'}[30m]))
                /
                sum(sum_over_time(pipelinerun_duration_scheduled_seconds_count{status='succeded'}[30m]))
            )
            /
            (
                sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_sum{status='success'}[30m]))
                /
                sum(sum_over_time(tekton_pipelines_controller_pipelinerun_duration_seconds_count{status='success'}[30m]))
            )  > 0.05
```